Playmate/View Controllers/Friends Related/InvitationsViewController.m

```diff
@@ -0,0 +1,109 @@
+//
+//  InvitationsViewController.m
+//  Playmate
+//
+//  Created by Jocelyn Tseng on 7/22/22.
+//
+
+#import "InvitationsViewController.h"
+#import "InvitationCell.h"
+#import "UIScrollView+EmptyDataSet.h"
+#import "Constants.h"
+#import "Helpers.h"
+#import "SessionDetailsViewController.h"
+#import "Session.h"
+
+@interface InvitationsViewController () <UITableViewDelegate, UITableViewDataSource, DZNEmptyDataSetSource, DZNEmptyDataSetDelegate, InvitationCellDelegate>
+
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (nonatomic, strong) NSArray *invitationsList;
+
+@end
+
+@implementation InvitationsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    self.tableView.rowHeight = 132;
```

Comment:
maybe add this to your constants file under a InvitationsViewController section, same for similar cases